src/ui/src/highlighterset.cpp

```diff
@@ -465,7 +465,7 @@ void HighlighterSetCollection::saveToStorage( QSettings& settings ) const
     settings.endGroup();
 }
 
-void HighlighterSetCollection::retrieveFromStorage( QSettings& settings )
+void HighlighterSetCollection::retrieveFromStorage(QSettings& settings )
```

Comment:
minor, but please stick to file convention, i.e. space after/before opening/closing bracket